_sass/button.scss

```diff
@@ -52,8 +54,11 @@ button {
         @include link-button;
         overflow: visible;
         border: 0;
-        color: $contentColor;
-        background: $fillColor;
+        color: $black;
+        background: $white;
+		border: 1px solid #dedede;
```

Comment:
Please use Variables instead of colour code.
https://github.com/ChampionFX/binary-style/blob/gh-pages/_sass/colors.scss